### ISSUE TITLE
Production runtime error when using baseAssets property

### ISSUE THREAD

**Issue by mmorgalocabal** [brillout/vite-plugin-ssr#775]
### Description

I recently made a major update from v0.4.62 to v0.4.112, and now I get an error on production when using the `baseAssets` property like this:
```
 // vite.config.js
 ssr({
    prerender: {
      partial: true
    },
    // Use BunnyCDN for assets on production
    baseAssets: env.NODE_ENV === 'production' ? 'https://job-blocks-production.b-cdn.net/' : undefined
  })
```
  
Using:
- "vite": "^4.2.1",
- "vite-plugin-ssr": "^0.4.112",

### Error Message + Error Stack

```shell
Error: [vite-plugin-ssr@0.4.112][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Go to https://github.com/brillout/vite-plugin-ssr/issues/new and copy-paste this error. (The error's stack trace is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Don't hesitate to reach out as it makes vite-plugin-ssr more robust.
    at removeFileExtentionAndHash (/mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/createHttpResponseObject.js:158:24)
    at isFontFallback (/mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/createHttpResponseObject.js:152:25)
    at /mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/createHttpResponseObject.js:37:17
    at Array.forEach (<anonymous>)
    at createHttpResponseObject (/mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/createHttpResponseObject.js:35:16)
    at async renderPageContext (/mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/renderPageContext.js:67:30)
    at async renderPageAttempt (/mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage.js:158:36)
    at async renderPage (/mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage.js:41:35)
    at async renderRoute (/mnt/DATA/jobsrepublic/projects/job-blocks/server/index.js:170:23)
```

**Comment by brillout** [comment]
If you remove the line `158` of `/mnt/DATA/jobsrepublic/projects/job-blocks/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/createHttpResponseObject.js`, does everything work then?

**Comment by mmorgalocabal** [comment]
That's right, it works with that line commented out.

**Comment by brillout** [comment]
Released as `vite-plugin-ssr@0.4.112-draft.36322`. (It may take a while until `vite-plugin-ssr@0.4.113` is released.)

Btw. in case your company is up for it: https://github.com/sponsors/brillout. Any (one-time) amount is appreciated!

**Comment by mmorgalocabal** [comment]
Thanks for the quick fix!
